src/lib/intent/editorinterface-update.ts

```diff
@@ -0,0 +1,44 @@
+import Intent from './base-intent'
+import { PlanMessage } from '../interfaces/plan-message'
+import { UpdateEditorInterfaceAction } from '../action/editorinterface-update'
+import chalk from 'chalk'
+import { SaveEditorInterfaceAction } from '../action/editorinterface-save'
+
+export default class EditorInterfaceUpdateIntent extends Intent {
+  isEditorInterfaceUpdate () {
+    return true
+  }
+  groupsWith (other: Intent): boolean {
+    const sameContentType = other.getContentTypeId() === this.getContentTypeId()
+    return other.isEditorInterfaceUpdate() && sameContentType
+  }
+  endsGroup (): boolean {
+    return true
+  }
+  shouldSave (): boolean {
+    return false
+  }
+  shouldPublish (): boolean {
+    return false
+  }
+  toActions () {
+    // TODO: check for optimizing API calls and saving CT by CT with merged editor interface definition
```

Comment:
Is this possible? It is its on API call, never seen it merged into a ContentType update

Reply:
Nah, I wanted ideally to merge editor interface changes of the same content type in a single API call, but that doesn't seem to be easy to do.
Maybe as a follow up!